.textlintrc

```diff
@@ -0,0 +1,544 @@
+{
+	"rules": {
+		"terminology": {
+			// Load default terms (see terms.json in the repository)
+			"defaultTerms": false,
+			// Syntax elements to skip. Overrides the default
+			"skip": [
+				"Blockquote"
+			],
+			// List of terms
+			"terms": [
+				// Brands
+				"Airbnb",
```

Comment:
This is an upstream carry-over which is itself an upstream (or well source) carryover.

This part of the list contains brand names to ensure they're properly capitalized. It could be trimmed, though obviously the thing is they won't be caught if they aren't in the config. So while they might not be applicable at the moment that doesn't mean they won't be in the future.